### PR TITLE
Add descriptions to get propagated into quarkus-universe-bom json

### LINKED ISSUE #9706: quarkus-universe-bom json contains duplicated description
**Description**
quarkus-universe-bom json contains duplicated description 
- "Build parent to bring in required dependencies"
- "Trace your services with Jaeger"

### PR DESCRIPTION
**Pull request by rsvoboda**
Add descriptions to get propagated into quarkus-universe-bom json

Fixes https://github.com/quarkusio/quarkus/issues/9706

### CODE REVIEW

**Comment by rsvoboda** [comment]
I'm touching only `<description>` element in pom.xml files, no test failures can be related to my changes.

**Review by gastaldi — APPROVED** [review]

File: extensions/vertx-graphql/runtime/pom.xml [hunk 1]
```diff
@@ -12,6 +12,7 @@
 
     <artifactId>quarkus-vertx-graphql</artifactId>
     <name>Quarkus - Quarkus - Vert.x GraphQL - Runtime</name>
+    <description>Query REST services using GraphQL</description>
```
Comment (michalszynkiewicz) [inline extensions/vertx-graphql/runtime/pom.xml:15]: @rsvoboda isn't GraphQL a different approach than REST?

Comment (rsvoboda) [reply extensions/vertx-graphql/runtime/pom.xml:15]: yeah, `Query the API using GraphQL` would be better I guess, I will create followup PR

Comment (rsvoboda) [reply extensions/vertx-graphql/runtime/pom.xml:15]: PR is here: https://github.com/quarkusio/quarkus/pull/9814

Thanks for spotting this

**Pull request merged**